chromium_src/net/http/transport_security_state.cc

```diff
@@ -0,0 +1,11 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "build/branding_buildflags.h"
+#undef BUILDFLAG_INTERNAL_GOOGLE_CHROME_BRANDING
```

Comment:
Usually we copy the entire #includes section from the original file so that the branding redefinition doesn't accidentally affect them.

Reply:
Added.